### PR TITLE
Adds 3 new nuclear related achievements

### PR DESCRIPTION
**Pull request by AsV9**
Winning as nukie, humiliating defeat, and suiciding with the disk

Tested with a database, so I know the logic in the `handle_achievements()` proc works.

:cl:  
rscadd: Added three new nuclear related achievements.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
time to greytide into caps office and suicide!

File: code/datums/achievements/achievements.dm [hunk 1]
```diff
@@ -22,3 +22,18 @@
 	name = "Catastrophe"
 	desc = "Emag a Particle Accelerator"
 	id = 4
+  
+/datum/achievement/flukeops
+	name = "Reverse Card"
+	desc = "As a member of the Crew deal a Humiliating defeat to Nuclear Team"
```
Comment (Amelia0010) [inline code/datums/achievements/achievements.dm]:
```suggestion
	desc = "As a member of the Crew, deal a Humiliating defeat to Nuclear Team"
```

File: code/datums/achievements/achievements.dm [hunk 2]
```diff
@@ -22,3 +22,18 @@
...
+	id = 5
+
+/datum/achievement/nukewin
+	name = "Delta Alert"
+	desc = "As a Nuclear Operative score a Major or Minor Victory"
```
Comment (Amelia0010) [inline code/datums/achievements/achievements.dm]:
```suggestion
	desc = "As a Nuclear Operative, score a Major or Minor Victory"
```

File: code/datums/achievements/achievements.dm [hunk 3]
```diff
@@ -22,3 +22,18 @@
...
+	id = 6
+
+/datum/achievement/honorarynukie
+	name = "Honorary Nukie"
+	desc = "Kill yourself with the nuclear authentication disk"
```
Comment (Amelia0010) [inline code/datums/achievements/achievements.dm]:
```suggestion
	desc = "Kill yourself using the nuclear authentication disk"
```

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**